streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/core/graph/executiongraph/ExecutionGraph.java

```diff
@@ -39,6 +40,11 @@
    */
   private long buildTime;
 
+  /**
+   * A monotonic increasing number, used for vertex's id(immutable).
+   */
+  private AtomicInteger lastExecutionVertexIndex = new AtomicInteger(0);
```

Comment:
I prefer using `vertexIdGenerator`

Reply:
Change name to 'executionVertexIdGenerator'. Also, use 'id' instead of 'globalIndex' in ```ExecutionVertex```.